### PR TITLE
React & Redux group project - Space Travelers' Hub

### PR DESCRIPTION
**Pull request by evans22j**
In this milestone;
- Create routes and view components (Rockets, Missions, My Profile)
- Create directories for all Redux state slice files (rockets, missions)
- Render UI:lists
- Fetch data and update Redux store
- Write actions and reducers for booking rockets and joining missions
- Render UI: conditional components rendering
- Render UI: My Profile section

### CODE REVIEW

File: src/pages/MyProfile.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+          </span>
+        ) : (
+          <span className="nothing"><span>You have not joined any missions</span></span>
+        )}
+      </div>
+      <div>
+        <h2>My Rockets</h2>
+      </div>
+    </div>
+  );
```
Comment (RolandM99) [inline src/pages/MyProfile.js]: - [ ] Please, Kindly add the reserved Rockets in your profile page. For this project, you should render rockets reserved and missions joined on your `My Profile` page. for now, I can see just missions joined but not the rockets reserved. Kindly fix this.

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi  @evans22j  and @L2oukmane , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💯 
- Good commit message ✔️ 
- Equal Commits Contributions ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/MyProfile.js (hunk 1 above)
Comment (peterrobert) [reply src/pages/MyProfile.js]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @evans22j, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**